<?xml version="1.0"?>
<sdf version="1.4">
  <world name="default">
    <include>
      <uri>model://ground_plane</uri>
    </include>
    <include>
      <uri>model://sun</uri>
    </include>

    <model name="ping_pong_table">
      <include>
        <uri>model://ping_pong_table_no_net</uri>
      </include>
    </model>

    <model name="net">
        <pose>0 0 .76 0 0 0 </pose>
        <include>
            <uri>model://ping_pong_table_net</uri>
        </include>
    </model>

    <model name="ping_pong_ball">
      <pose>0.6 0 1.25  0 0 0</pose>
      <include>
        <uri>model://ping_pong_ball</uri>
      </include>
      <plugin name="ball_plugin" filename="libball_plugin.so"/>
    </model>

    <plugin name="training_return_plugin" filename="libtraining_return_plugin.so"/>
  </world>
</sdf>
